avocado/plugins/jsonresult.py

```diff
@@ -81,6 +81,11 @@ def _render(result):
         return json.dumps(content, sort_keys=True, indent=4, separators=(",", ": "))
 
     def render(self, result, job):
+        LOG_UI.warning(
+            "WARNING: The result.json test attributes related to time has been renamed in version 104 "
```

Comment:
I've run jobs with this warning, and I believe the informational aspect is not worth the "nagging", because very few users will actually care about this.

I would prefer, a lot, if this warning is removed completely.